### PR TITLE
Replace TravisCI with GitHub actions

### PR DESCRIPTION
**Pull request by kylemh**
https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing

Seen other projects make the switch, and I figured we'd be forced to eventually anyways

### CODE REVIEW

**Comment by kylemh** [comment]
@zimme I'd like to use https://github.com/codecov/codecov-action but don't have the ability to add secrets to the repo. Will you do the honors?

Specifically: `CODECOV_TOKEN`

File: .github/workflows/release.yml
Comment (hamzahamidi) [inline .github/workflows/release.yml:19]: can you change it to npm ci instead ?

File: .github/workflows/ci.yml
Comment (hamzahamidi) [inline .github/workflows/ci.yml:21]: same thing with npm ci

File: .github/workflows/release.yml
Comment (hamzahamidi) [inline .github/workflows/release.yml:23]: Won't you need to pass an npm token in your env variables? Not sure

**Review by hamzahamidi — COMMENTED** [review]
Exactly we're forced to migrate to GitHub actions. However we still need @zimme to add the an npm token to publish as a secret to the repository.

File: .github/workflows/release.yml
Comment (kylemh) [reply .github/workflows/release.yml:19]: we dont have a lockfile, so I don't think this would work, right? I suppose I could add one!

File: .github/workflows/release.yml
Comment (kylemh) [reply .github/workflows/release.yml:23]: Neither was I. Tough to check on this one until after it's merged 😂

File: .github/workflows/release.yml
Comment (hamzahamidi) [reply .github/workflows/release.yml:19]: Ok let's keep an npm i then.

**Review by hamzahamidi — APPROVED** [review]

File: .github/workflows/release.yml
Comment (zimme) [reply .github/workflows/release.yml:23]: I've added a `NPM_TOKEN` secret to both this repo and prettier-eslint-cli

**Comment by zimme** [comment]
Added a CODECOV_TOKEN secret to this repo and cli repo as well.

**Comment by kylemh** [comment]
@zimme can you add NPM_TOKEN as well? That'll do it for me. Thank you!

**Comment by zimme** [comment]
I turned off travis-ci.org for this repository just now.

**Comment by zimme** [comment]
Ping me when/if you make the same migration for the `prettier-eslint-cli` repo and I'll turn off travis-ci.org for that repo too.

**Comment by kylemh** [comment]
I wasn't aware I had access to `prettier-eslint-cli`, do I?

**Comment by zimme** [comment]
I believe you should, I was given admin access a while back and added you and the others that wanted to help out in the maintainer issue.

**Pull request merged**
